### PR TITLE
[SEIS-1172] Add pre-stack trace header overrides

### PR DESCRIPTION
**Pull request by skairunner**
Depends on #207

### CODE REVIEW

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 1]
```diff
@@ -63,6 +63,8 @@ message SegyOverrides {
     google.protobuf.Int32Value cdp_number_offset = 6;              // Position of the ensemble (CDP) number in trace headers. Defaults to 21 as per the SEG-Y rev1 specification
     google.protobuf.Int32Value energy_source_point_offset = 7;     // Position of the energy source point in trace headers. Defaults to 17 as per the SEG-Y rev1 specification
     google.protobuf.Int32Value shotpoint_offset = 8;               // Position of the shotpoint field in trace headers. Defaults to 197 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value cdp_trace_offset = 9;               // Position of the trace number within ensemble. Defaults to 21 as per the SEG-Y rev1 specification.
+    google.protobuf.Int32Value distance_from_source_offset = 10;   // Position of the distance from source ("offset"). Defaults to 37 as per the SEG-Y rev1 specification.
```
Comment (audunska) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:67]: Seems unfortunate to deviate from the standard here, given that we chose not to in the other place.

What about `offset_header_offset`?

Comment (skairunner) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:67]: 👍

**Review by audunska — APPROVED** [review]
A bit of a funny name, but it works I guess

**Comment by skairunner** [comment]
> A bit of a funny name, but it works I guess

I will always prefer `distance_from_source`/`offset` in the bottom of my heart 😢

**Pull request merged**